### ISSUE TITLE
Generator fails when it encounters numeric value with decimal point

### ISSUE THREAD

**Issue by kfrajtak** [brechtm/rinohtype#251]
I have a bunch of RST documents created from DOCX documents with `pandoc`. 

Images are transformed to entries like the following
```
.. |image17| image:: media/image20.png
   :width: 6.41528in
   :height: 6.00208in
```

and it fails with
```
Exception occurred:
  File "c:\python39\lib\site-packages\rinoh\frontend\rst\nodes.py", line 668, in convert_quantity
    value, unit = RE_LENGTH_PERCENT_UNITLESS.match(quantity_string).groups()
AttributeError: 'NoneType' object has no attribute 'groups'
```

This is my temporal fix at https://github.com/brechtm/rinohtype/blob/master/src/rinoh/frontend/rst/nodes.py#L648 that works:
```
RE_LENGTH_PERCENT_UNITLESS = re.compile(r'^(?P<value>\d+\.?\d*)(?P<unit>[a-z%]*)$')
```

**Comment by brechtm** [comment]
Thanks for the bug report and suggested fix! This will be included in the 0.5.3 release.

**Comment by kfrajtak** [comment]
I am glad I could help.
